benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1256,6 +1256,8 @@
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingDate", "UserRole": "citizen", "CRUD": "CRUD"},
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "CaseFieldID": "workBasketHearingEpimsId", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
```

Comment:
I'm not sure why the others are in the prod sheets, could you confirm whether this should be in as part of the S&L release?

Reply:
Doesn't the PET team make changes directly to prod?

Reply:
After discussion, this should be a nonprod change, so this change will be reverted and applied to the relevant nonprod sheet

Reply:
Relevant changes applied. Resolving this conversation.